test/new-e2e/pkg/e2e/suite_utils.go

```diff
@@ -20,3 +30,73 @@ func (tl testLogger) Write(p []byte) (n int, err error) {
 	tl.t.Log(string(p))
 	return len(p), nil
 }
+
+// GetRootOutputDir creates and returns a directory for tests to store output files and artifacts.
+// A timestamp is included in the path to distinguish between multiple runs, and os.MkdirTemp() is
+// used to avoid name collisions between parallel runs.
+//
+// A new directory is created on each call to this function, it is recommended to save this result
+// and use it for all tests in a run. For example see BaseSuite.GetTestOutputDir().
+//
+// See runner.GetProfile().GetOutputDir() for the root output directory selection logic.
+//
+// See GetTestOutputDir for a function that returns a subdirectory for a specific test.
```

Comment:
<!-- Suggestions propose improvements to the current subject. It’s important to be explicit and clear on what is being suggested and why it is an improvement. -->
**💬 suggestion**
I would move the creation part to the suite function that is called once, leaving the formatting to this static function that can be called multiple times from elsewhere.

Reply:
When moving, we should be explicit with the title with the creation part. It can be misleading to have a directory created to a `Get` function

Reply:
> It can be misleading to have a directory created to a Get function
👍 

I don't think we can separate the name formatting from the creation here, since `os.MkdirTemp` creates the dir with a random/unique pattern. I could rename the function to `CreateRootOutputDir`.

Reply:
Yes I like it!